src/lib.rs

```diff
@@ -177,6 +177,7 @@ impl SocketBuilder {
 
     /// Sets the target namespace of the client. The namespace must start
     /// with a leading `/`. Valid examples are e.g. `/admin`, `/foo`.
+    #[deprecated(note = "use namespace instead")]
```

Comment:
We might write 'use `namespace(nsp)` instead' in order to indicate that this refers to a function.

Reply:
```suggestion
     #[deprecated(note = "use namespace(nsp) instead")]
```